<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyList</class>
 <widget class="QWidget" name="KeyList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="menuWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="switchContextButton">
            <property name="toolTip">
             <string>Uncheck ALL</string>
            </property>
            <property name="text">
             <string>Switch Context</string>
            </property>
            <property name="icon">
             <iconset resource="../gpgfrontend.qrc">
              <normaloff>:/icons/database.png</normaloff>:/icons/database.png</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="channelLcdNumber">
            <property name="smallDecimalPoint">
             <bool>true</bool>
            </property>
            <property name="digitCount">
             <number>1</number>
            </property>
            <property name="mode">
             <enum>QLCDNumber::Mode::Dec</enum>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::SegmentStyle::Filled</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="refreshKeyListButton">
            <property name="toolTip">
             <string>Refresh</string>
            </property>
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset resource="../gpgfrontend.qrc">
              <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="syncButton">
            <property name="toolTip">
             <string>Sync Public Key</string>
            </property>
            <property name="text">
             <string>Sync Public Key</string>
            </property>
            <property name="icon">
             <iconset resource="../gpgfrontend.qrc">
              <normaloff>:/icons/cloud_sync.png</normaloff>:/icons/cloud_sync.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="checkALLButton">
            <property name="toolTip">
             <string>Check ALL</string>
            </property>
            <property name="text">
             <string>Check ALL</string>
            </property>
            <property name="icon">
             <iconset resource="../gpgfrontend.qrc">
              <normaloff>:/icons/total_selection.png</normaloff>:/icons/total_selection.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="uncheckButton">
            <property name="toolTip">
             <string>Uncheck ALL</string>
            </property>
            <property name="text">
             <string>Uncheck ALL</string>
            </property>
            <property name="icon">
             <iconset resource="../gpgfrontend.qrc">
              <normaloff>:/icons/none.png</normaloff>:/icons/none.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="columnTypeButton">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../gpgfrontend.qrc">
              <normaloff>:/icons/filter.png</normaloff>:/icons/filter.png</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="keyGroupButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../gpgfrontend.qrc">
              <normaloff>:/icons/key-group.png</normaloff>:/icons/key-group.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="searchBarEdit">
            <property name="maxLength">
             <number>32</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="keyGroupTab">
       <property name="tabShape">
        <enum>QTabWidget::TabShape::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../gpgfrontend.qrc"/>
 </resources>
 <connections/>
</ui>
